html/janus.js

```diff
@@ -1064,10 +1086,10 @@ function Janus(gatewayCallbacks) {
 			},
 			error: function(textStatus, errorThrown) {
 				Janus.error(textStatus + ":", errorThrown);	// FIXME
-				if(Janus.isArray(servers) && !callbacks["reconnect"]) {
+				if(autoTryOtherServers && Janus.isArray(servers) && !callbacks["reconnect"]) {
```

Comment:
I think this is missing some condition here: my point in my previous comment was that we do want an automated fallback to other servers if we fail to connect entirely (that's the purpose of servers), but that should not happen if we managed to connect to a server, did something, and then lost connection. It looks like this new `autoTryOtherServers` property is conditionally changing the behaviour of the first case, and not dealing with the second one.

In both cases, it may be enough to check what the value of `connected` is, since we set it to `true` the moment we successfully connected to a server: if it's `true`, don't fallback (or maybe let's envisage a new property to allow a fallback for that specific case), while if it's false, do fallback (or see what `autoTryOtherServers` says).